src/visualization/visualize_treatment_models.py

```diff
@@ -1,5 +1,5 @@
 """
-Visual explanations of the models
```

Comment:
what do you think about allowing this function to take the parameter save the plot? It would be good to get our naming convention for saving plots figured out. Recall we talked about including the code version and date created. 

Also, we should be able to call this function from the colab notebook too so we don't repeat code.

Reply:
Yes, agreed, good point. Will make the change.